src/TemplateSkinManager.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Handle form template view skin.
+ *
+ * @package Give
+ */
+
+namespace Give;
+
+/**
+ * Class TemplateSkinManager
+ *
+ * @since 2.7.0
+ * @package Give
+ */
+class TemplateSkinManager {
```

Comment:
I don't know why, but referring to something as "skin" feels weird to me. Let's call it `IframeView` or something like that to also indicate its intended use.

Let's also move this out of the root. It should belongs with the view classes.

Reply:
Got it. I also thought that but waiting to listen from you.

Reply:
resolved